airbnb_project/listings/views.py

```diff
@@ -42,23 +52,17 @@ def run_spider_in_thread(runner):
 def harvest_listings(request):
     """
     Django view to initiate the harvesting process.
-
-    This view starts a Scrapy spider to harvest listings. It ensures that
-    only one harvesting process runs at a time and handles various error
-    scenarios.
-
-    Args:
-        request (HttpRequest): The incoming request object.
-
-    Returns:
-        HttpResponse: A response indicating the status of the harvesting process.
     """
+    global is_scraping
+    print("harvesting_listing", is_scraping)
```

Comment:
Do we want this `print` ?

Reply:
This is a debug print left in the code, should be deleted.

Reply:
thank you!